includes/Handler/class-jsonld.php

```diff
@@ -45,8 +45,8 @@ public function parse( Response $response, $target_url ) {
 		}
 
 		// Set raw data.
-		$this->webmention_item->set_raw( $jsonld );
-		$this->webmention_item->set_response_type( 'mention' );
+		$this->webmention_item->add_raw( $jsonld );
```

Comment:
Why change the name from set to add? Is there some reason?

Reply:
Currently every handler that is used overwrites the informations from the handler before, because he 'sets' the new data without checking if they already exist. The `add` functions are calling the `set` functions in the end, but with a check if they are already set.